src/cf-buttons/src/atoms/buttons.less

```diff
@@ -169,4 +170,14 @@
       padding-bottom: unit( @btn-v-padding-modifier-ie * 15px / @btn__super-font-size, em );
     }
   }
+
+  //
+  // Full width button on x-small screens
+  //
+  &__full-on-xsmall {
+    .respond-to-max( @bp-xs-max, {
+        display: block;
+        width: 100%;
```

Comment:
As much as it pains me to say it, adjust indentation on these two lines to match the two-space indentation in the rest of the file.